api/v1/Base.php

```diff
@@ -88,10 +89,20 @@ public static function Error($d): void
     // phpcs:ignore PSR1.Methods.CamelCapsMethodName.NotCamelCaps
     public static function ErrorString($errorException): string
     {
-        return $errorException->getMessage() . PHP_EOL . "Of file " . $errorException->getFile() . " On line " .
+        $errorException = self::getOriginalError($errorException);
```

Comment:
Always show the orginal error. We might want to show the whole exception tree later.... if needed